src/wp-includes/functions.wp-scripts.php

```diff
@@ -121,13 +121,14 @@ function wp_print_scripts( $handles = false ) {
  *
```

Comment:
```suggestion
 * @since 6.3.0 Added `$standalone` boolean parameter.
 *
```

Reply:
@mukeshpanchal27 I've added this for now, but there is still the ongoing conversation of whether we keep or discard the concept of standalone inline scripts.